### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by BeatLink** [Jimmy-Z/grub-iso-boot#2]
Would it be possible to add windows support?

**Comment by Jimmy-Z** [comment]
Impossible to support, read the "some explanation" section in readme, it's mostly the distributions' work, not this script, in case of Windows, I doubt Microsoft would ever support this.

I'll keep this not closed intentionally.

**Comment by BeatLink** [comment]
I see. Thank you. It would be better to let others know through the main README

**Comment by Jimmy-Z** [comment]
Good point, but I think it was clear enough since the description says "... various Linux ..."

**Comment by BeatLink** [comment]
Ahh i see. In any case, I'm working on a GTK GUI based on your script! Readme is a bit outdated tho. Its a python app so just setup a venv, install pygobject and try it out! I'm going to package it for flatpak and appimage and pypi when i find the time. Im also gonna do a gui counterpart to ventoy.

https://github.com/BeatLink/UMDC

Edit: After discovering Ventoy, I've decided to focus on creating a GUI for that script. UMDC will still be up and functional but the project will be archived. Everything works, all that is needed is packaging for PyPI and Flatpak

Edit 2: Even better, ill just transfer the repo to you @Jimmy-Z if you want. Feel free to keep or delete as you see fit.
